contrib/zpages/README.md

```diff
@@ -50,6 +50,37 @@ public class MyMainClass {
 }
 ```
 
+### View stats and spans on Z-Pages
+
+#### View RPC stats on /rpcz page
+
+The /rpcz page displays the canonical RPC stats broken down by RPC methods. Example:
+
+![rpcz-example](https://user-images.githubusercontent.com/10536136/37544216-74f04012-2921-11e8-8fa2-3d5d00edd400.png)
```

Comment:
Shouldn't the images be stored in the repository, so that the readme and images are synchronized?

Reply:
Done. People may want to host images elsewhere and add a superlink to it to make the git repo smaller, but I think these images should be fine (984k, while our repo is already 8.6m).